lib/govuk_app_config/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukAppConfig
-  VERSION = "9.8.0".freeze
+  VERSION = "9.9.0".freeze
```

Comment:
Nitpick: we could make this `9.8.1`, because it's not a breaking change (insofar as no other apps will have made any changes in light of the change introduced in 9.8.0) and it's more of a bugfix than a new feature.

Reply:
Thanks @ChrisBAshton, will update :+1: